mkdocs.yml

```diff
@@ -70,7 +70,6 @@ nav:
       - Haber Hardfork(BSC): ./announce/haber-bsc.md
       - Haber Hardfork(opBNB): ./announce/haber-opbnb.md
       - Second Sunset Hardfork(BC testnet): ./announce/second-sunset-bc-testnet.md
-
```

Comment:
Only this change, miss something?

Reply:
Seems all other commits have been added to the main branch already... that is the reason.